js/newArticle.js

```diff
@@ -1,21 +1,43 @@
+var makeNewArticle = {};
+var articlePreview = {};
+var cheese = {};
```

Comment:
My fault for when we were messing around with your code, but this should be changed back into something that is a better descriptor of what it actually represents.